proto/src/lib.rs

```diff
@@ -4,4 +4,8 @@
 
 pub mod serializers;
 
+#[cfg(not(feature = "tonic"))]
 include!(concat!(env!("OUT_DIR"), "/mod.rs"));
+
+#[cfg(feature = "tonic")]
+::tonic::include_proto!("mod");
```

Comment:
are those outputs compatible for the types that are common to them? I'm wondering if it's safe to mix crates that depends on celestia-proto with and without the tonic feature

Reply:
Good question, I believe so as in our crates compile for both tonic and non-tonic. I'll try to get a project with mixed tonic-ness and test that too.

Reply:
Since we use `include_file("mod.rs")` in prost and tonic builders, it is safe to just use `include!(concat!(env!("OUT_DIR"), "/mod.rs"));` for both of them.

 Also common types are compatible, because tonic for generates type definitions with prost and then generates additional code for the grpc services.

Reply:
Yes, I didn't comment to confirm back, but the code is mostly the same, with additional traits implemented. 

Also true, we can use `include!` in both, I was mostly looking for an excuse to use nicer looking tonic version :P Rolled back.